spec.html

```diff
@@ -4522,6 +4522,7 @@ <h1>
         <dl class="header">
         </dl>
         <emu-alg>
+          1. If _size_ > 2<sup>53</sup> - 1, throw a *RangeError* exception.
```

Comment:
This AO is only called by `AllocateSharedArrayBuffer`, which in turn is only called by `SharedArrayBuffer()` after `ToIndex`—and `ToIndex` already throws for values not in the range <code>0 ≤ integer ≤ 2<sup>53</sup> - 1</code>. This should at best be an assertion. The only way to get a larger-than-expected array buffer, AFAICT, is through the `TypedArray()` constructor => passing a `length` number => `AllocateTypedArray` => `AllocateTypedArrayBuffer` => `AllocateArrayBuffer`.

Reply:
Thank you for the catch, I wanted to add the new step into `CreateByteDataBlock`, but accidentally added it to `CreateSharedByteDataBlock` instead... That should be fixed now.